reporter/src/funTest/assets/notice-template-reporter-expected-results

```diff
@@ -88,35 +88,6 @@ Package: @ort:declared-license:1.0
 
 The following copyrights and licenses were found in the source code of this package:
 
-Copyright 1
```

Comment:
The existing test for the notice template reporter did not check the evaluation of the property that controls the inclusion of a specific license. I extended the test, so that this property is now *true* for one license and *false* for another. This is the reason why the expected results are changed.

Reply:
Thanks for the explanation. Could you please also make this more clear in the commit message, using the wording of your comment here?

Reply:
Updated the commit message.